### DISCUSSION TITLE
`/webhooks` page not available

### DISCUSSION THREAD

**Discussion by amjadsaadeh** [firefly-iii/firefly-iii#7863] [Firefly III] [answered]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

According to the [webhook docs](https://docs.firefly-iii.org/firefly-iii/features/webhooks/) ist should be possible to access webhook management by calling `/webhooks` page. But I get an 404 page instead.

### Debug information

Debug information generated at 2023-08-17 16:10:30 Europe/Berlin for Firefly III version **6.0.20**.

| Scope | Version |
| --- | --- |
| Firefly III | 6.0.20 |
| Firefly III API | 2.0.6 |
| Build | 597, 17-08-2023 10:06:07 UTC |
| Base Build | 35, 30-07-2023 03:52:37 UTC |
| DB version | 20 (exp. 20) |
| PHP | `8.2.8` |
| Host | `Linux` |

| System info | Value |
| --- | --- |
| System TZ | Europe/Berlin |
| Browser TZ | Europe/Berlin |
| App environment | local |
| App debug mode | false |
| App cache driver | file |
| App logging | info, stack |
| Display errors | Off |
| Error reporting | ALL errors |
| Max upload | 67108864 (64 MB) |
| Interface | apache2handler |
| Default language | en_US |
| Default locale | equal |
| BCscale | 12 |
| DB drivers | mysql, pgsql, sqlite |
| Current driver | pgsql |
| Trusted proxies (.env) | `**` |

| User info | Value |
| --- | --- |
| Login provider | eloquent  |
| User guard | web  |
| Headers | N/A, N/A  |
| Stateful domains |  |
| Session start |  |
| Session end |  |
| Session first |  |
| User ID | 1 |
| User language | de_DE |
| User locale | de_DE |
| Attempt at "de_DE.utf8" | 'de_DE.utf8' |
| Attempt at "de_DE.UTF-8" | 'de_DE.UTF-8' |
| User agent | Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/115.0.0.0 Safari/537.36 Edg/115.0.1901.203 |

### Expected behaviour

A valid page to manage webhooks.

### Steps to reproduce

1. Call Firefly III url
2. Append `/webhooks` and hit Enter

### Additional info

I used the helm chart to install it in a k8s environment, but used the 6.0.20 image.

**Comment by JC5** [comment, marked as answer]
Did you enable it in your environment variables? It is disabled by default.

**Reply by amjadsaadeh** [reply]
Yes I missed the environment variable, sorry.

**Comment by JC5** [comment]
Found it. The documentation is wrong. Try `/webhooks/index`.

**Reply by amjadsaadeh** [reply]
In fact, IT was both :)

**Comment by Deeschen** [comment]
I'm not sure how to even start on webhooks can someone briefly explain it to me? I'm trying to make a discord webhook

**Reply by JC5** [reply]
Sorry this is not really a generic helpdesk, if you have specific Firefly III questions I can help of course but otherwise it's up to you.
